packages/editor/src/store/selectors.js

```diff
@@ -353,11 +354,12 @@ export function isEditedPostSaveable( state ) {
 	//  See: <PostPublishButton /> (`forceIsDirty` prop)
 	//  See: https://github.com/WordPress/gutenberg/pull/4184
 
-	return (
+	const isEditedPostSavable = (
```

Comment:
Everywhere else (including the function itself) we use the spelling "saveable"

Reply:
old habits die hard